etc/profile-a-l/joplin.profile

```diff
@@ -0,0 +1,54 @@
+# Firejail profile for joplin
+# Description: A note taking and to-do application
+# This file is overwritten after every install/update
+quiet
+
+# Persistent local customizations
+include joplin.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}/wayland-*
+blacklist ${RUNUSER}
+
+include allow-bin-sh.inc
+include allow-nodejs.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+noblacklist /usr/share/joplin
+mkdir ${HOME}/.config/joplin
+whitelist ${HOME}/.config/joplin
+whitelist ${HOME}/.config/nano
+whitelist ${HOME}/.emacs
+whitelist ${HOME}/.emacs.d
+whitelist ${HOME}/.nanorc
+whitelist ${HOME}/.vim
+whitelist ${HOME}/.viminfo
+whitelist ${HOME}/.vimrc
```

Comment:
Add `include whitelist-common.inc`

Reply:
is this needed.
whitelist-common.inc seems looks to be more oriented for ui.

Reply:
yes